[2024-03-20 08:57:30,631] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: elt_reddit_pipeline.copy_to_redshift manual__2024-03-20T08:57:24.388901+00:00 [queued]>
[2024-03-20 08:57:30,637] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: elt_reddit_pipeline.copy_to_redshift manual__2024-03-20T08:57:24.388901+00:00 [queued]>
[2024-03-20 08:57:30,638] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2024-03-20 08:57:30,638] {taskinstance.py:1357} INFO - Starting attempt 1 of 2
[2024-03-20 08:57:30,638] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2024-03-20 08:57:30,645] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): copy_to_redshift> on 2024-03-20 08:57:24.388901+00:00
[2024-03-20 08:57:30,649] {standard_task_runner.py:52} INFO - Started process 1593 to run task
[2024-03-20 08:57:30,652] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'elt_reddit_pipeline', 'copy_to_redshift', 'manual__2024-03-20T08:57:24.388901+00:00', '--job-id', '8', '--raw', '--subdir', 'DAGS_FOLDER/elt_reddit_pipeline.py', '--cfg-path', '/tmp/tmpiqmpyprw', '--error-file', '/tmp/tmp_9fz7mxq']
[2024-03-20 08:57:30,653] {standard_task_runner.py:80} INFO - Job 8: Subtask copy_to_redshift
[2024-03-20 08:57:30,691] {task_command.py:370} INFO - Running <TaskInstance: elt_reddit_pipeline.copy_to_redshift manual__2024-03-20T08:57:24.388901+00:00 [running]> on host 9aae7e0ec976
[2024-03-20 08:57:30,763] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=elt_reddit_pipeline
AIRFLOW_CTX_TASK_ID=copy_to_redshift
AIRFLOW_CTX_EXECUTION_DATE=2024-03-20T08:57:24.388901+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-03-20T08:57:24.388901+00:00
[2024-03-20 08:57:30,764] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2024-03-20 08:57:30,765] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'python /opt/***/extraction/upload_aws_redshift_etl.py 20240320']
[2024-03-20 08:57:30,776] {subprocess.py:85} INFO - Output:
[2024-03-20 08:57:30,908] {subprocess.py:92} INFO - Unable to connect to Redshift. Error could not translate host name ""redshift-cluster-pipeline.cuz9cskwdwtb.ap-northeast-2.redshift.amazonaws.com"" to address: Name or service not known
[2024-03-20 08:57:30,909] {subprocess.py:92} INFO - 
[2024-03-20 08:57:30,918] {subprocess.py:96} INFO - Command exited with return code 1
[2024-03-20 08:57:30,940] {taskinstance.py:1889} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/bash.py", line 195, in execute
    f'Bash command failed. The command returned a non-zero exit code {result.exit_code}.'
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 1.
[2024-03-20 08:57:30,945] {taskinstance.py:1400} INFO - Marking task as UP_FOR_RETRY. dag_id=elt_reddit_pipeline, task_id=copy_to_redshift, execution_date=20240320T085724, start_date=20240320T085730, end_date=20240320T085730
[2024-03-20 08:57:30,960] {standard_task_runner.py:97} ERROR - Failed to execute job 8 for task copy_to_redshift (Bash command failed. The command returned a non-zero exit code 1.; 1593)
[2024-03-20 08:57:30,992] {local_task_job.py:156} INFO - Task exited with return code 1
[2024-03-20 08:57:31,031] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
